Automattic/woocommerce-payments
# Update fraud status logic on payment details

Fixes #6017 

#### Changes proposed in this Pull Request

Add `order` data to the WC_Payments_API_Intention class to be used on the payment details page. We're currently using the `latestFraudOutcome` of a given payment intent to determine its correct status.









#### Testing instructions

- Checkout to #5888 
- Follow the test prep step
- Jump to the "Order held for review with expired auth" test and follow the steps
- Click on the "Review payment" link in the "Fraud meta box"
- The payment status will appear as "Payment blocked"
- Checkout to this branch
- Refresh the page
- The payment status will appear as "Refunded"





-------------------


**Post merge**
